sdk/search/azure-search/src/main/java/com/azure/search/implementation/SearchIndexRestClientBuilder.java

```diff
@@ -0,0 +1,144 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+// Code generated by Microsoft (R) AutoRest Code Generator.
+// Changes may cause incorrect behavior and will be lost if the code is
+// regenerated.
+
+package com.azure.search.implementation;
+import com.azure.core.util.serializer.SerializerAdapter;
+
+import com.azure.core.annotation.ServiceClientBuilder;
+import com.azure.core.http.HttpPipeline;
+import com.azure.core.http.HttpPipelineBuilder;
+import com.azure.core.http.policy.CookiePolicy;
+import com.azure.core.http.policy.RetryPolicy;
+import com.azure.core.http.policy.UserAgentPolicy;
+
+/**
+ * A builder for creating a new instance of the SearchIndexRestClient type.
+ */
+@ServiceClientBuilder(serviceClients = SearchIndexRestClientImpl.class)
```

Comment:
Builders and clients in `implementation` package don't have to be annotated with `@ServiceClientBuilder` or `@ServiceClient`

Reply:
This is a generated class, the annotation is injected by autorest